Sources/SwifterSwift/SwiftStdlib/OptionalExtensions.swift

```diff
@@ -40,6 +40,21 @@ public extension Optional {
         return wrapped
     }
 
+    /// SwifterSwift: Gets the wrapped value of an optional. If the optional is `nil`, calls Never-returning closure.
+    ///
+    ///        let foo: String? = nil
+    ///        foo.or { preconditionFailute() } -> Throws preconditionFailure with default description
```

Comment:
change this `preconditionFailute()` to `preconditionFailure()`. Just a small typo.